lib/screens/widgets/map.widget.dart

```diff
@@ -115,21 +112,22 @@ class MapWidgetState extends State<MapWidget> {
               child: FlutterMap(
                 mapController: mapController,
                 options: MapOptions(
-                  center: homeLocation ?? addressLocations.values.first,
-                  zoom: 13.0,
                   maxZoom: 17,
                   minZoom: 3,
-                  interactiveFlags:
-                      InteractiveFlag.pinchZoom | InteractiveFlag.drag,
+                  initialCameraFit: CameraFit.bounds(
+                    bounds: bounds,
+                    padding: const EdgeInsets.all(32),
+                  ),
+                  interactionOptions: const InteractionOptions(
+                      flags: InteractiveFlag.pinchZoom | InteractiveFlag.drag),
                 ),
                 children: [
                   TileLayer(
                     // TODO: Recommended: Do not hardcode any URL to tile.openstreetmap.org as doing so will limit your ability to react if the service is disrupted or blocked. In particular, switching should be possible without requiring a software update.
                     urlTemplate:
-                        'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-                    subdomains: const ['a', 'b', 'c'],
+                        'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
```

Comment:
I've got a warning in the console, that the usage of the subdomains is deprecated and should just be removed.